[OKTA-1174908]: Milestone#1: OIN refactoring for "Build an SSO integration" (#6155)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/create-an-app-integration/main/index.md

### Create a custom app integration

You can add an app integration that doesn't exist in the OIN, using the App Integration Wizard (AIW). The wizard allows you to create a custom app integration for your app and connect Okta with your SAML, OIDC, SWA, or SCIM app. You can also add SCIM provisioning to a custom app integration.

>**Note**: As a best practice, create two or three extra admin users in your Okta org to manage the integration. This ensures that your team can access the integration for updates in the future.

1. Open the Admin Console for your org.

> 1. Go to **Applications > Applications**.

Reviewer: ```suggestion
1. Go to **Applications** > **Applications**.
```

Once your app is set up:

1. Assign users:

    a. Click the **Assignments** tab.

    b. Click **Assign** and then select either **Assign to People** or **Assign to Groups**.

>     c. Enter the appropriate people or groups that you want to have Single Sign-On into your app, click **Assign** for each, verify user attributes, and select **Save and Go Back**.

Reviewer: ```suggestion
    c. Enter the appropriate people or groups that you want to have SSO into your app, click **Assign** for each, verify user attributes, and select **Save and Go Back**.
```

    d. Click **Done**.

1. Test SSO: Sign in to the Okta End-User Dashboard as a test user and click the app tile.

    >**Note:**. This step applies only to SSO (OIDC or SAML) integrations.

> 1. Verify redirection: Ensure the app-initiated flow correctly redirects back from the Okta sign-in page to your app.

Reviewer: ```suggestion
1. Verify redirection: Ensure that the app-initiated flow correctly redirects back from the Okta sign-in page to your app.
```
